### PR TITLE
Correctly coerce inf to infinity

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

**Comment by stefanv** [comment]
@dannygoldstein Any ideas for a test here?

**Review by acrellin — APPROVED** [review]

**Comment by dannygoldstein** [comment]
@stefanv See proposed changes on https://github.com/stefanv/skyportal/pull/2

File: skyportal/handlers/api/photometry.py [hunk 1]
```diff
@@ -370,6 +370,11 @@ def _compile_photometry_values(element, compiler, asfrom=False, **kw):
 
             def coerced_value(elem, column):
                 literal_value = compiler.render_literal_value(elem, column.type)
+                if literal_value == 'inf':
+                    literal_value = '+infinity'
```
Comment (dannygoldstein) [inline skyportal/handlers/api/photometry.py:374]: I think this needs to be string-escaped 
```suggestion
                    literal_value = "'infinity'"
```

**Review by dannygoldstein — CHANGES_REQUESTED** [review]
See suggested changes on https://github.com/stefanv/skyportal/pull/2

**Comment by stefanv** [comment]
Failures unrelated.

**Comment by stefanv** [comment]
The content of this patch is now by @dannygoldstein, and @acrellin has approved, so I will go ahead and merge (failures are unrelated).

**Pull request merged**